tests/views/conftest.py

```diff
@@ -4,3 +4,10 @@
 def api_client():
     from rest_framework.test import APIClient
     return APIClient()
+
+@pytest.fixture
+def player():
+    from mars_api.models import Player
+    p = Player(nickname="my nickname", motto="a short phrase")
+    p.save()
```

Comment:
adding the fixture is good but 
![](https://media.giphy.com/media/9G3wg7lH5DpxC/giphy.gif)